### PR TITLE
Expand community data

### LINKED ISSUE #155: Ability to upload pdf for claim
## Context
* At the very moment users can only upload images when making a claim. 

## Issue
* Allow users to upload pdfs for claim. 
* Ensure to put a limit on file size

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes #155 

## Changes Proposed ( a list of new changes introduced by this PR)
- [x] Add `memberCount` to `communityPreview` query
- [x] Add photos for the community
- [x] Add website to the community
- [x] Allows PDF to be uploaded
- [x] Update Phoenix to 1.5

## How to test ( a list of instructions on how to test this PR)
- `mix test`
- Check if communityPreview has a memberCount field and it is correct 
- Make sure its possible to add photos to the community, and only the creator can do it
- Make sure graphql allows you to have a website on a community and communityPreview

### CODE REVIEW

File: lib/cambiatus_web/controllers/upload_controller.ex [hunk 1]
```diff
@@ -1,14 +1,17 @@
 defmodule CambiatusWeb.UploadController do
+  @moduledoc false
+
   use CambiatusWeb, :controller
 
-  alias Cambiatus.Upload
+  alias Cambiatus.Upload.Uploader
```
Comment (venomnert) [inline lib/cambiatus_web/controllers/upload_controller.ex:6]: Should be renamed to `Cambiatus.File.Uploader`

File: test/cambiatus/upload_test.exs [hunk 2]
```diff
@@ -3,6 +3,7 @@ defmodule Cambiatus.UploadTest do
   Unit tests for the upload module
   """
   use Cambiatus.DataCase
+  alias Cambiatus.Upload.Uploader
```
Comment (venomnert) [inline test/cambiatus/upload_test.exs:6]: Should be renamed to `Cambiatus.File.Uploader`

**Review by venomnert — CHANGES_REQUESTED** [review]

**Comment by venomnert** [comment]
@lucca65 this PR also closes issue #155

**Comment by lucca65** [comment]
@venomnert since you are a organization member, you can edit the description of any PR, please feel free to update this one as well 💪

**Review by venomnert — CHANGES_REQUESTED** [review]

File: lib/cambiatus/commune/community.ex [hunk 3]
```diff
@@ -56,4 +61,14 @@ defmodule Cambiatus.Commune.Community do
...
+
+  def save_photos(%Community{} = community, urls) do
+    uploads = Enum.map(urls, &%Upload{url: &1})
+
+    community
+    |> Repo.preload([:uploads, :subdomain])
+    |> changeset(%{})
+    |> Ecto.Changeset.put_assoc(:uploads, uploads)
+    |> Repo.update()
+  end
```
[the remark this answers is not in the document]
Comment (lucca65) [reply lib/cambiatus/commune/community.ex:73]: hey bro, the behavior is intended!

[the remark this answers is not in the document]
Comment (venomnert) [reply lib/cambiatus/commune/community.ex:73]: Thanks for the confirmation!

**Review by venomnert — APPROVED** [review]

**Pull request merged**